### PR TITLE
Krhubert/database

### PR DESCRIPTION
**Pull request by NicolasMahe**

### CODE REVIEW

**Comment by NicolasMahe** [comment]
I approve this PR

File: database/services/save.go [hunk 1]
```diff
@@ -5,18 +5,17 @@ import (
...
-		return
+		return "", err
 	}
 	db, err := open()
 	defer close()
 	if err != nil {
-		return
+		return "", err
 	}
 	hash = service.Hash()
```
Comment (antho1404) [inline database/services/save.go]: to stay consistent with the result of the refactoring, I would not use the named output and assign a new variable here

Comment (MarinX) [reply database/services/save.go]: ...but the whole package consists of named outputs.
Should we refactor service package to remove named outputs?

Comment (antho1404) [reply database/services/save.go]: hum no not really, let's do things step by step, i'm ok with that anyway

**Pull request merged**